transmissions.go

```diff
@@ -47,9 +47,9 @@ type TxOptions struct {
 	TmplOptions
```

Comment:
According the Go's [doc guide](https://blog.golang.org/godoc-documenting-go-code) Paragraph 5&6 the comment should start with `TxOptions ` instead of `Options`